inc/class_parser.php

```diff
@@ -1332,6 +1343,17 @@ function mycode_parse_img_disabled_callback4($matches)
 		return $this->mycode_parse_img_disabled($matches[5]);
 	}
 
+	/**
+	 * Parses IMG MyCode disabled.
+	 *
+	 * @return string Raw image code notice.
+	 */
+	function mycode_parse_img_disabled_callback5()
+	{
+		global $lang;
+		return $lang->posted_image_uri;
```

Comment:
Shouldn't this behave the same as the other disabled images by calling `mycode_parse_img_disabled`?